### DISCUSSION TITLE
Using Firebase Give The Error

### DISCUSSION THREAD

**Discussion by SatyamDevv** [Sketchware-Pro/Sketchware-Pro#310] [Q&A] [answered]
storage/emulated/0/.sketchware/mysc/775/bin/res/project.zip: error: Invalid file.
error: failed parsing overlays.

I am using Firebase First Time I am getting this error again and again I Try to use In different projects but I get this error my android version is 7 and Sketchware version is latest 

I also try to delete that file and run again but it still give this

**Comment by j-px** [comment, marked as answer]
Could you try to build your project again, and when it fails, upload /Internal storage/.sketchware/debug.txt here?
